src/js/shade.js

```diff
@@ -92,6 +93,7 @@
                     opacity: this.options.opacity
                 },
                 {
+                    queue: false,
```

Comment:
It's actually a good change in general — it fixes the animation jank when shade was animating at a different duration than pinny/pikabu. It's necessary for pikabu though.